.travis.yml

```diff
@@ -1,5 +1,5 @@
-sudo: false
 language: node_js
+sudo: false
```

Comment:
Please move the `sudo` back to the top -- it's a personal preference to declare how Travis is running first =/